### DISCUSSION TITLE
Backlog of planning concerns

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#29] [General]
# Backlog of planning considerations

We have published [a list of planning considerations](https://digital-land.github.io/data-standards/what-we-are-working-on/backlog). This is a list of things we've heard about or people have told us about.

We plan to work through this list, exploring each thing and seeing if creating and opening up data will solve some users' needs.

If you think we are missing something, please let us know. Tell us:

* what we are missing
* what you know about how it affects planning
* where you think the data would be used, and why

**Comment by mattlucht** [comment]
On a recent call with Historic England there was reference to a dataset around previously listed buildings (I think!)

**Comment by colmjude** [comment]
@mattlucht as in buildings that were on the list of [listed buildings](#10), and that have now been removed?

**Comment by mattlucht** [comment]
@colmjude yes, although there's every chance that I misunderstood! If that is the case, maybe it implies that Listed Building records are removed from the dataset

**Comment by LanePlanning** [comment]
Development Consent Order boundary is missing from the data set. Work need to switch from shapefile to multi polygon but should be part of the planning database as orders are made under Planning Act 2008 for nationally significant infrastructure projects and contain power to develop and to restrict activities as well as temporary possession, compulsory acquisition and management of rights of access etc. See https://infrastructure.planninginspectorate.gov.uk/ for made and draft orders.
Pauleen.Lane.kc@planninginspectorate.gov.uk

**Comment by mattlucht** [comment]
> Development Consent Order boundary is missing from the data set. Work need to switch from shapefile to multi polygon but should be part of the planning database as orders are made under Planning Act 2008 for nationally significant infrastructure projects and contain power to develop and to restrict activities as well as temporary possession, compulsory acquisition and management of rights of access etc. See https://infrastructure.planninginspectorate.gov.uk/ for made and draft orders. [Pauleen.Lane.kc@planninginspectorate.gov.uk](mailto:Pauleen.Lane.kc@planninginspectorate.gov.uk)

thanks @LanePlanning i'll get this added to the backlog now so that we can keep track of it. Maybe we can arrange a quick chat so that I can understand a bit more about this? (matt.lucht1@levellingup.gov.uk)

**Comment by colmjude** [comment]
We have been reviewing the [backlog of planning considerations](https://digital-land.github.io/data-standards/what-we-are-working-on/planning-consideration/), which has increased to 335 items.

The review highlighted 12 we think are duplicates.

We propose to record the different names used for a planning consideration and archive these items. The items are

| To archive | Duplicate of |
| ----------- | ----------- |
| Air quality management locations | Air quality management area (AQMA) |
| COMAH – major hazard sites | Control of major accident and hazard sites |
| Geology 625K | BGS Geology 625k |
| Land ownership boundary | Land registry polygons / red line boundaries |
| Nuclear safety zones | Nuclear zones |
| Protected sites (statutory sites) | Duplicate of a few, including SAC, SPA, Ramsar, Proposed Ramsar, SSSI, Marine Conservation Area, Local Nature Reserve, Local Wildlife SIte, National Parks and the Broads, AONB, Heritage Coast |
| Public transport access nodes | National public transport access nodes (NaPTAN) |
| Safeguarded aerodromes | Safeguarded military explosives site |
| Safety hazard areas | Health and Safety Executive Consultation (HSE) Zones |
| Section 73 | Planning application variance |
| Watercourses | Duplicate of a few including Canals, Lake, Main Rivers, Reservoirs etc |
| Wellmaster hydrogeological database index | Borehole scans |

If you disagree with our assessment please speak up and let us know why you think it is different to the planning consideration we have paired it with.

**Comment by colmjude** [comment]
We have been doing another review of [the backlog of planning considerations](https://digital-land.github.io/data-standards/what-we-are-working-on/planning-consideration/). It was getting pretty long - we've been throwing everything and anything on there for a couple of years now and had over 350 things to look at. 

We thought it would make sense to "archive" what we can to have a narrower list to focus on. We will archive things that we felt were outside the current remit of planning.data.gov.uk, things we aren't sure what they are, and things we felt were old/outdated. We will be left with a list we are more confident will provide value once the items are added to [planning.data.gov.uk](https://www.planning.data.gov.uk/).

The items will only be archived, so if there is a need for them, they can easily come back.

The items we'll be archiving are:

| Planning consideration |
| ---------------------- |
| Advertisements
| BGS geological indicators of flooding |
| BGS geological map scans |
| BGS Geology 10k |
| BGS Geology 250k |
| BGS Geology 25k |
| BGS Geology 50k |
| BGS Geology 625k |
| BGS Geology: superficial thickness |
| BGS groundwater flooding |
| BGS mineral resources |
| BGS permeability |
| Borehole scans |
| CASI multispectral imagery |
| Central government buildings |
| Contravention of building notices |
| Crossrail route |
| Crossrail stations |
| Dune slack likely locations |
| Environmental pollution incidents |
| Explosive and ordnance storage - Ordnance |
| Food hygiene rating |
| Food safety notice |
| GeoClimate |
| Geological disturbance - faults |
| Geological disturbance - fissure and breakline |
| GeoScour |
| GeoSure 5km hex grids |
| GeoSure shrink-swell 3D |
| GeoSure shrink-swell subsurface |
| Houses in multiple occupation |
| Hospital performance |
| Hydrogeological maps |
| Inner urban improvement area |
| Key business areas |
| LIDAR composite DSM - 2m |
| LIDAR composite DTM - 1m |
| LIDAR composite DTM - 25cm |
| Major railway improvement schemes |
| Mine entry |
| Mineral assessment reports |
| Mining and ground stability |
| MORECS |
| Multibeam bathymetry |
| National coach services |
| National landslide database |
| Nearby railway schemes |
| Nearby road schemes |
| Noise abatement |
| Notices, Orders, Directions under Planning Acts |
| Outstanding notices |
| Price paid data |
| Probable coal mining |
| Public health service delivery performance |
| Public transport timetables |
| Rainfall data |
| Recorded flood outlines |
| Regional flood and coastal committees |
| Renewable energy planning database (REPD) |
| Retail opportunity areas |
| Risk of flooding from rivers and seas – postcodes |
| Risk of flooding from rivers and seas – properties |
| Risk of flooding from surface water |
| River corridor features |
| Roadways, footways and footpaths |
| Rocky shores |
| Shallow coal mine workings |
| SOBI (Single Onshore Boreholes Index) |
| Soil parent material model |
| Soil type |
| Species data |
| TOID |
| Town centre boundaries |
| Traffic schemes |
| Transaction data |
| Transport (under TCPA route) |
| Traveline national dataset |
| UK house price data |
| Urban development area |
| Vertical aerial photography |
| Waterside regeneration zones |
| Weather forecasting |
| Wellmaster hydrogeological database index |
| Wildlife |

Please let us know if there is anything we should keep.

**Reply by colmjude** [reply]
These items have now been [archived](https://digital-land.github.io/data-standards/what-we-are-working-on/planning-consideration/archived/).

**Reply by colmjude** [reply]
Another set of items we plan to archive are those we don't think are proper planning terms. We expect that if they are a thing we should look at they will have a different name, a name planners use.

They are:

| Planning consideration |
| ---------------------- |
| Areas for environmental enhancement |
| Built up areas |
| Landmark buildings |
| Landmark features |
| London squares |
| Long established & protected woodland |
| Millennium greens |

**Reply by colmjude** [reply]
These items have now been [archived](https://digital-land.github.io/data-standards/what-we-are-working-on/planning-consideration/archived/).

**Reply by colmjude** [reply]
We are also planning to archive these items - they are things more in the statistical space so we are not sure they are a good fit for [planning.data.gov.uk](https://www.planning.data.gov.uk/). If you disagree then let us know.

They are:

| Planning consideration | Notes |
| ---------------------- | ----- |
| Energy and renewable statistics | |
| Hazard waste interrogator | |
| Housing waiting list | We think this could be covered in housing target numbers as part of the housing need. |
| National flood and coastal erosion risk management strategy | |
| Waste and recycling statistics | |

**Reply by colmjude** [reply]
These have now all been archived.

**Comment by clairemiller2** [comment]
We are aiming to reduce instances of duplicate entries in our [backlog of planning considerations](https://digital-land.github.io/data-standards/what-we-are-working-on/planning-consideration/), while ensuring that the planning considerations we do have are clear in what they cover.

Many local planning authorities safeguard land to protect it from specific types of development, or to limit the extent of development. This is often done through the Local Plan process.

Examples of safeguarded land include:
- Sometimes it is necessary for major infrastructure schemes to safeguard land from alternate development uses (e.g. HS2)
- Mineral safeguarding areas
- Safeguarded aerodrome, technical sites and military explosives storage areas

As this term is broad and can cover a range of types of safeguarded land, we propose to remove this from the backlog, and instead have specific datasets (e.g. minerals safeguarded land). If we were to remove this overarching safeguarded land dataset, what specific forms of safeguarded land would you need on the platform?

**Comment by stevenjmesser** [comment]
Tewkesbury Borough Council have identified a need to use the [Zone of Influence for Cotswold Beechwoods](https://tewkesbury.gov.uk/wp-content/uploads/2024/12/cotswold-beechwoods-zoi-map.pdf) special area of conservation. Extra charges are levied on users in this area. More information: https://tewkesbury.gov.uk/services/planning/biodiversity-wildlife/cotswold-beechwoods-special-area-of-conservation-sac/

They would like to be able to use the area in a PlanX flow. I have not looked at whether Natural England supply this data but other special areas of conservation have zones of influence too.

**Comment by Stef-Webb** [comment]
The Department for Energy Security and Net Zero (DESNZ) are developing a policy for [heat network zones](https://www.gov.uk/government/publications/heat-network-zoning-overview).  Heat Networks Zones are geographical locations where heat networks have been modelled to be the lowest cost, lowest carbon solution for decarbonising heat.  New developments in heat network zones will be mandated to connect to the heat network and as such could be considered a planning consideration.

The legal framework for heat network zoning is established under the Energy Act 2023, which received Royal Assent in October 2023. This Act grants the Secretary of State the authority to implement heat network zoning across England through secondary legislation, the drafting of statutory instruments to enable this is underway.
